packages/iconset/package.json

```diff
@@ -64,6 +64,7 @@
     },
     "types": "./src/index.d.ts",
     "customElements": "custom-elements.json",
+    "deprecationNotice": "@spectrum-web-components/iconset is deprecated and it will be removed in the future release.",
```

Comment:
Can we match this with a Dev Mode warning? I love the idea of the docs site notices, and I want to live in a world where devs read documentation, but I do not read documentation, so come to where I work.

Reply:
It should already be there.
https://github.com/adobe/spectrum-web-components/blob/df4f2a4464dcc484f60c15cd88bcc2f167b32022/packages/iconset/src/iconset.ts#L26